### PR TITLE
feat(NerdGraph): Log data obfus

### PR DESCRIPTION
**Pull request by barbnewrelic**
Log data obfuscation placeholder for DOC-7721

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+
+You can use NerdGraph at [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql) to create, query, and manage your data obfuscation rules for logs. NerdGraph is our GraphQL-format API explorer.
+
+**Reviewer:** What about the ability to set up obfuscation rules in the UI? I saw a front end design doc in Atlassian. Is there an ETA for that? Will both the API and the UI options be released at the same time?
+
+## Obfuscation and hashing [#definitions]
+
+Obfuscation is when we entirely mask or remove data with no means for retrieval. For example, if a log line `Sent 123-45-6789` becomes `Sent XXX-XX-XXXX`, that's obfuscation.
+
+Hashing is when a sensitive value is replaced with that value, sent through a one-way hashing algorithm. This allows lookup of related records if you have the original value and are able to hash it. For instance, if a log line `Sent 123-45-6789` is hashed to `Sent XYZAB`, the `XYZAB` is the calculated result of hashing `123-45-6789`.
```
Comment (jsubirat) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx:23]:
```suggestion
*Hashing* is when a sensitive value is replaced using a one-way hashing algorithm. This allows lookup of related records if you have the original value and are able to hash it (our UI provides a simple tool to do this, for your convenience). For instance, if a log line `Sent 123-45-6789` is hashed to `Sent 01a54629efb952287e554eb23ef69c52097a75aecc0e3a93ca0855ab6d7a31a0`, the `01a54629efb952287e554eb23ef69c52097a75aecc0e3a93ca0855ab6d7a31a0` is the calculated result of hashing `123-45-6789`.
```

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+
+* Field-level obfuscation or hashing applies to the entire attribute on an event.
+* In-message obfuscation or hashing means the rules can target part of a message, such as the `Sent` message examples in this section.
```
Comment (jsubirat) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx:26]: Do we need to mention this? The users will be defining their own obfuscation expressions, so they will be the ones in charge of specifying whether to hash/mask the whole attribute, or to just hash/mask a part of it. Probably we don't need to mention this, then?

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+---
+
+You control what customer data is logged with our log management service, so be sure to follow your organization's security guidelines to mask, obfuscate, or prevent sending any sensitive data. Our log management service automatically masks number patterns that appear to be for items such as credit cards or Social Security numbers. For more information, see our [security documentation](/docs/logs/get-started/new-relics-log-management-security-privacy) for log management.
+
+We also provide options for you to manage and sanitize parts of your customer data when it arrives at New Relic. This makes it easier and safer for you, rather than needing to track down sensitive data in all of your applications and try to fix them at the source.
+
+You can use NerdGraph at [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql) to create, query, and manage your data obfuscation rules for logs. NerdGraph is our GraphQL-format API explorer.
+
+**Reviewer:** What about the ability to set up obfuscation rules in the UI? I saw a front end design doc in Atlassian. Is there an ETA for that? Will both the API and the UI options be released at the same time?
+
```
Comment (jsubirat) [inline src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx:18]: [⚠️] Please remember to remove this! Initially (MVP), we will not allow the set up of obfuscation rules in the UI, only via the API. As part of the GA, though, we plan on adding support for this in the UI as well. Not sure about the official GA date, though.

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx (hunk 2 above)
Comment (barbnewrelic) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx:26]: agreed. thanks for pointing this out!

File: src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx (hunk 3 above)
Comment (barbnewrelic) [reply src/content/docs/apis/nerdgraph/examples/nerdgraph-log-data-obfuscation.mdx:18]: will do. thanks for confirming!

**Comment by barbnewrelic** [comment]
Replaced by https://github.com/newrelic/docs-website/pull/6391

**Pull request closed without merging**